/*
* @Copyright: Copyright[2015]<Alejandro Bordallo>
* @Date:      2015-05-12
* @Email:     alex.bordallo@ed.ac.uk
* @Desc:      ROS wrapper for Nao's fsr sensors
*/
#ifndef FSR_H
#define FSR_H

#include <ros/ros.h>

#include "definitions.h"

using namespace std;

class Fsr {
 public:
  Fsr();
  ~Fsr();

  // ROS publishers

 private:
  // ROS

  // NaoQI

};

#endif  /* FSR_H */
